src/views/screens/details/template.njk

```diff
@@ -129,6 +135,12 @@
 
 {% block bodyEnd %}
   {{ super() }}
+  <script>
+    var isDetails = true;
+    var coordinates = {{ docDetails.geographicBoundary | safe }};
+    var center = "{{ docDetails.geographicCenter }}";
```

Comment:
this could break. docDetails may be undefined, you have the condition checked at line 57. do a safe reading and set a default value or render it inside the if condition

Reply:
This has been resolved now.